src/NexusMods.App.UI/Pages/Sorting/LoadOrder/LoadOrderView.axaml

```diff
@@ -66,30 +64,42 @@
             </Border>
 
             <StackPanel Grid.Row="1" Spacing="24" Margin="24">
-                <alerts:Alert
-                    Severity="Info"
-                    Title="Load Order for REDmod files in Cyberpunk 2077 - First Loaded Wins"
-                    Body="Some Cyberpunk 2077 mods use REDmod files to alter core gameplay elements. If two REDmod files modify the same part of the game, the one loaded first will take priority and overwrite changes from those loaded later.\n\nFor example, the 1st position overwrites the 2nd, the 2nd overwrites the 3rd, and so on."
-                    IsVisible="True"
-                    ShowDismiss="False" />
-
-                <TextBlock Text="Last Loaded REDmod File Wins"
-                           Theme="{StaticResource HeadingXSSemiTheme}" />
+                <alerts:Alert x:Name="LoadOrderAlert"
+                              Severity="Info"
+                              Title="{Binding InfoAlertTitle}"
+                              Body="{Binding InfoAlertMessage}"
```

Comment:
Will need to remember to change these to Code behind bindings

Reply:
I agree. We will need more bindings as Laurence has added more tooltips etc so will move to code behind when we go through and tidy up the data. Next week now. Also need to refactor all the styling out into files as well.